scripts/src/report/verifier_report.py

```diff
@@ -171,7 +223,7 @@ def validate(report_path):
             v1_0_profile = False
             if profile_version.major == 1 and profile_version.minor == 0:
                 v1_0_profile = True
-        except Exception:
+        except ValueError:
```

Comment:
1. Do we want to ignore other exceptions types?
2. Are we certain the Version.coerce call that's likely to through an error here is going to throw a ValueError?